Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,110 @@
+   - the longest line generated depends on how many assigned-crypto-domains and assigned-cryptos are found
+   - minimum data set characteristics
+     - RECFM: VB
+     - LRECL: 800
+
+## Invocation
+**Syntax**:
+```
+  RXCRYPT1 -D outputDataSet [-C CPCname] [-S Status] [-I] [-V]                                                
+ ```
+ where:
+  - *-D outputDataSet*
+      - **required**
+      - *outputDataSet* is the name of an existing partitioned data set
+      - if the query is successful, a member containing the crypto
+        information in a CSV format will be stored into the specified data set: *outputDataSet(memberName)* where the name of data set member is:
+        - LOCAL, if the LOCAL CPC was used
+        - CPCname, if a specific CPC was specified via the -C option
+  - *-C CPCname*
+      - **optional**
+      - *CPCname* is the name of the CPC which hosts the LPARs whose corresponding image activation profile crypto info will be retrieved
+      - **default if not provided:** LOCAL CPC
+  - *-S Status*
+      - **optional**
+      - *Status* represents the status of the LPAR and is an optional filter for the list of LPARs. The names associated with these LPARs is used as input for which image activation profiles need to be queried.
+      - Valid values:          
+        - operating - query profiles for activation image profiles
+        associated with active/operating LPARs  
+        - not-operating - query profiles for activation image profiles     associated with active/not operating LPARs                                                  
+        - not-activated - query profiles for activation images profiles
+        associated with not-activated LPARs 
+      - **default if not provided or if not valid:** operating                                               
+  - *-I*
+      - **optional**
+      - indicates this exec is being run in an ISV REXX environment
+      - **default if not set:** TSO/E REXX environment
+  - *–V*
+      - **optional**
+      - indicates additional tracing associated with JSON parsing should be turned on
+      - **default if not set:** tracing excludes JSON specific tracing
+
+**sample invocation in TSO:**
+<br>RXCRYPT1 has been copied into data set HWI.HWIREST.REXX and HWI.RXCRYPTO.OUTPUT
+    has been allocated as RECFM=VB, LRECL=800
+```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT'
+```
+ - exec is running in a TSO/E rexx environment and will query the crypto information associated with LPARs in operating status on the LOCAL CPC
+```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT -C CPC1'
+```
+ - exec is running in a TSO/E rexx environment and will query the crypto information associated with LPARS in operating status on CPC CPC1
+ ```
+ex 'HWI.HWIREST.REXX(RXCRYPT1)' '-D HWI.RXCRYPTO.OUTPUT -C CPC1 -S not-activated'
+```
+ - exec is running in a TSO/E rexx environment, will query the crupton information associated with LPARs in not-activated status on CPC CPC1
```

Comment:
typo: 'crupton' should be 'crypto'

Reply:
fixed, lol, can't tell you how many times I typed "crypton" !!